uptane-standard.md

```diff
@@ -867,6 +867,7 @@ In order to perform full verification, an ECU SHALL perform the following steps:
 11. For each image listed in the Targets metadata file from the Director repository, locate a Targets metadata file that contains an image with exactly the same file name. For each delegated Targets metadata file that is found to contain metadata for the image currently being processed, perform all of the checks in step 10. Use the following process to locate image metadata:
     1. If the top-level Targets metadata file contains signed metadata about the image, return the metadata to be checked and skip to step 11.3.
     2. Recursively search the list of delegations, in order of appearance:
+        0. If the delegation does not address the image name being sought -- if none of the delegation's wildcard paths match this image name -- skip this delegation.
```

Comment:
Is there some way to phrase this to include other ways to list images besides wildcards (enumerated list, etc)?

Reply:
Could we also start from 1 instead of 0, please?

Reply:
Yeah, but currently, renumbering it would be a source of conflicts with the other PRs touching the later items in this section.  After?

Reply:
@mnm678 Did we land on a way to phrase that at some point?  Patterns?  Path lists?  I agree that that's important.  If there isn't a consensus already, maybe patterns?  That does still suggest wildcards.  Suggestions otherwise?

Reply:
I don't remember if we agreed on something, but earlier they are referred to as 'image paths'

Reply:
Switched to "image paths", then.